pkg/s3/controllerserver.go

```diff
@@ -149,10 +149,15 @@ func (cs *controllerServer) ValidateVolumeCapabilities(ctx context.Context, req
 		return nil, status.Error(codes.NotFound, fmt.Sprintf("Volume with id %s does not exist", req.GetVolumeId()))
 	}
 
-	for _, cap := range req.VolumeCapabilities {
-		if cap.GetAccessMode().GetMode() != csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER {
-			return &csi.ValidateVolumeCapabilitiesResponse{Supported: false, Message: ""}, nil
-		}
-	}
-	return &csi.ValidateVolumeCapabilitiesResponse{Supported: true, Message: ""}, nil
+	// TODO: checkout if this code is still needed?
```

Comment:
We still need this as we can only support ReadWriteOnce volumes: https://github.com/container-storage-interface/spec/blob/master/lib/go/csi/csi.pb.go#L136

Like this we are just returning that we support everything the orchestrator has requested.

Reply:
I added a function to master to sanitize the volume name before creating a bucket so if you rebase on master the tests should be fine. So if you could solve the above I will merge the pull request. It would be great if we could do this soon as I'm a bit blocked in implementing other things before this.